libs/interpret-text/src/lib/TextExplanationDashboard/CommonUtils.ts

```diff
@@ -78,6 +78,17 @@ export class Utils {
     return sortedList;
   }
 
+  public static addItem(value: number, radio: string | undefined): boolean {
+    if (
```

Comment:
What if we just return this condition instead of explicit true/false? If `radio` is undefined, then this condition may anyway return false.

Reply:
+1

Reply:
thank you, updated